### PR TITLE
added onBlur event in datepicker for final-form

### PR DESCRIPTION
**Pull request by raphaelblum**

### CODE REVIEW

**Comment by raphaelblum** [comment]
onblur wird benötigt für final-form:

https://stackoverflow.com/questions/51776786/with-react-final-form-why-is-meta-touched-always-false-with-third-party-componen

File: packages/react-admin-date-picker/src/DatePicker.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const DatePicker: React.FC<IProps> = ({
     }, [localeName]);
 
     return (
-        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color}>
+        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color} onBlur={(event: React.FocusEvent<HTMLInputElement>) => onBlur(event)}>
```
Comment (rabengraph) [inline packages/react-admin-date-picker/src/DatePicker.tsx:36]:
```suggestion
        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color} onBlur={onBlur}>
```
.. wäre nicht so verbose

**Review by kaufmo — APPROVED** [review]

File: packages/react-admin-date-picker/src/DatePicker.tsx (hunk 1 above)
Comment (raphaelblum) [reply packages/react-admin-date-picker/src/DatePicker.tsx]: passt hab ich umgeändert

**Comment by nsams** [comment]
Build in master fails now with this merge, never merge failing PRs into master!

And now when looking at this diff, did you test this? I doubt onBlur on a div works - as a div can't have the focus.

**Pull request merged**